cmd/watch.go

```diff
@@ -102,6 +109,16 @@ func handleWatchEvent(client kit.ThemeClient, asset kit.Asset, event kit.EventTy
 		blue(asset.Key),
 	)
 	if err := perform(client, asset, event, nil); err != nil {
+		notificationMessage("There is a error in your templates check your logs", "Watch", "")
 		stdErr.Printf("[%s] %s", green(client.Config.Environment), err)
 	}
 }
+
+func notificationMessage(message string, title string, iconPath string) {
```

Comment:
Does this method need an iconPath if you never use it or pass it? Also if this is only used for watch then does it need the title param?

Reply:
There is no need to pass the param. I will remove that.